libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -146,11 +147,29 @@ const inferError = (
     maxAmountFrom: string;
     errorCode?: number;
     errorMessage?: string;
+    status?: string;
   }
 ): Error | CustomMinOrMaxError | undefined => {
   const tenPowMagnitude = new BigNumber(10).pow(unitFrom.magnitude);
-  const { amountTo, minAmountFrom, maxAmountFrom, errorCode, errorMessage } =
-    responseData;
+  const {
+    amountTo,
+    minAmountFrom,
+    maxAmountFrom,
+    errorCode,
+    errorMessage,
+    status,
+  } = responseData;
+
+  // DEX quotes are out of limits error. We do not know if it is a low or high limit, neither the amount.
+  if (
+    (!minAmountFrom || !maxAmountFrom) &&
+    status === "error" &&
+    errorCode !== 300
```

Comment:
@jnicoulaud-ledger What does the 300 error code mean and why do we use it? I see it's multiple choices but not sure why we use that. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/300#:~:text=The%20HTTP%20300%20Multiple%20Choices,more%20than%20one%20possible%20responses.

Reply:
We are receiving different kind of error from Paraswap and 1Inch, perhaps if we unify them we can make this part of the code cleaner.

Reply:
@ggilchrist-ledger it's not an HTTP status code, it's a "swap API error code" (and yes it sucks that we use error codes that look like HTTP status codes, we have a plan to change it): https://github.com/LedgerHQ/swap/blob/master/modules/model/src/main/scala/co/ledger/swap/model/ApplicationError.scala#L22

In the meanwhile there is a mapping here to avoid harcoding these codes: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/exchange/swap/index.ts#L198